packages/wpcom.js/Makefile

```diff
@@ -0,0 +1,85 @@
+# get Makefile directory name: http://stackoverflow.com/a/5982798/376773
```

Comment:
Do we need this file at all, if we're switching to `calypso-build`?

Reply:
The `Makefile` and `.gitignore` still contains some scripts information that needs to be converted to something else -- NPM build scripts in `package.json` and a better `.gitignore`. That's why I didn't delete them yet.

Reply:
Sounds good, let's keep them, then 👍